### ISSUE TITLE
[1.12.2] Conflict with HEI + IC2 : Game crashes if "ic2:fluid_cell" renders in HEI.

### ISSUE THREAD

**Issue by Z-Tunic** [LoliKingdom/LoliASM#123]
Minecraft: 1.12.2
Forge: 14.23.5.2860

Mods: 4
!mixinbooter-5.0
HadEnoughItems_1.12.2-4.22.2
loliasm-5.5
industrialcraft-2-2.8.221-ex112

Crash:
[crash-2022-07-24_13.38.40-client.txt](https://github.com/LoliKingdom/LoliASM/files/9176435/crash-2022-07-24_13.38.40-client.txt)

Log:
[latest.log](https://github.com/LoliKingdom/LoliASM/files/9176432/latest.log)

Debug:
[debug.log](https://github.com/LoliKingdom/LoliASM/files/9176434/debug.log)

---

I updated from 'loliasm-4.13' due to the animated textures being bugged and was gonna check if they were fixed with 'loliasm-5.5'. Now the game now crashes with Had Enough Items if Industrialcraft 2 is installed. If you try to view a fluid cell item in HEI... the game goes boom. Unsure if there are other items that are affected.

I could've posted this in HEI but seeing as this just happens after updating ASM and the game not crashing if ASM isn't installed... it lead me to think that the issue could be made here.

*EDIT*
I blacklisted the "ic2:fluid_cell" from being shown in HEI and after going through a list of items in my survival world... the game no longer crashes. There may be some mods out there that will crash though but at least this is a workaround for those using Industrialcraft 2.

...until you view recipes that use the ic2:fluid_cell, then the game will crash once more.

**Comment by embeddedt** [comment]
Other items are also affected, e.g. from Buildcraft.

```
---- Minecraft Crash Report ----
// Lolis deobfuscated this stacktrace using MCP's stable-39 mappings.
// Don't be sad, have a hug! <3

Time: 2022-07-24 15:46:00 EDT
Description: Rendering item

java.lang.NullPointerException
    at net.minecraft.client.renderer.RenderItem.handler$zhc000$beforeRenderItem(RenderItem.java:1173)
    at net.minecraft.client.renderer.RenderItem.renderQuads(RenderItem.java:226)
    at net.minecraftforge.client.ForgeHooksClient.drawSegment(ForgeHooksClient.java:754)
    at net.minecraftforge.client.ForgeHooksClient.renderLitItem(ForgeHooksClient.java:716)
    at net.minecraft.client.renderer.RenderItem.renderModel(RenderItem.java:118)
    at net.minecraft.client.renderer.RenderItem.renderModel(RenderItem.java:106)
    at net.minecraft.client.renderer.RenderItem.renderItem(RenderItem.java:149)
    at net.minecraft.client.renderer.RenderItem.renderItemModelIntoGUI(RenderItem.java:308)
    at net.minecraft.client.renderer.RenderItem.renderItemAndEffectIntoGUI(RenderItem.java:347)
    at codechicken.lib.render.item.CCRenderItem.renderItemAndEffectIntoGUI(CCRenderItem.java:356)
    at net.minecraft.client.gui.inventory.GuiContainer.drawSlot(GuiContainer.java:265)
    at net.minecraft.client.gui.inventory.GuiContainer.drawScreen(GuiContainer.java:98)
    at net.minecraft.client.renderer.InventoryEffectRenderer.drawScreen(InventoryEffectRenderer.java:51)
    at net.minecraft.client.gui.inventory.GuiContainerCreative.drawScreen(GuiContainerCreative.java:643)
    at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:396)
    at net.minecraft.client.renderer.EntityRenderer.updateCameraAndRender(EntityRenderer.java:1124)
    at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1119)
    at net.minecraft.client.Minecraft.run(Minecraft.java:5114)
    at net.minecraft.client.main.Main.main(SourceFile:123)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.polymc.impl.OneSixLauncher.invokeMain(OneSixLauncher.java:104)
    at org.polymc.impl.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:175)
    at org.polymc.impl.OneSixLauncher.launch(OneSixLauncher.java:185)
    at org.polymc.EntryPoint.listen(EntryPoint.java:144)
    at org.polymc.EntryPoint.main(EntryPoint.java:74)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Item being rendered --
  Item Type: buildcraft.transport.item.ItemPipeHolder@28a4b930
  Registry Name: buildcrafttransport:pipe_sandstone_item
  Item Aux: 0
  Item NBT: null
  Item Foil: false

-- Item being rendered --
  Item Type: buildcraft.transport.item.ItemPipeHolder@28a4b930
  Item Aux: 0
  Item NBT: null
  Item Foil: false

-- Screen render details --
  Screen name: net.minecraft.client.gui.inventory.GuiContainerCreative
  Mouse location: Scaled: (466, 180). Absolute: (933, 659)
  Screen size: Scaled: (960, 510). Absolute: (1920, 1019). Scale factor of 2
```

**Comment by Rongmario** [comment]
Null sprites... I'll fix it

**Comment by Rongmario** [comment]
Forgot to close. Fixed in 5.6.
